### PR TITLE
Calling correct backend endpoints

### PR DESCRIPTION
**Pull request by andrewconner**
@leogrim FYI

@cvializ

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileSlackUpsell.js [hunk 1]
```diff
@@ -17,31 +17,29 @@ angular.module('kifi')
...
+              messageTicker({ text: 'Oops, that didn’t work. Try again?', type: 'red' });
+            }
+          });
         };
 
         $scope.onClickedSyncAllSlackChannels = function() {
           var org = $scope.getOrg();
           $analytics.eventTrack('user_clicked_pane', { type: 'orgProfileSlackUpsell', action: 'syncAllChannels' });
-          $window.location = '/site/organizations/' + org.id + '/slack/sync/public';
+          slackService.publicSync($scope.getOrg().id).then(function (resp) {
```
Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileSlackUpsell.js:34]: why not use the `var org` above?

Comment (cvializ) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileSlackUpsell.js:34]: or remove it, either way

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/orgProfile/orgProfileSlackUpsell.js:34]: You beat me :)

**Comment by cvializ** [comment]
:sunglasses: :+1: :+1:

File: kifi-backend/modules/shoebox/angular/src/home/featureUpsell.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ angular.module('kifi')
         scope.me = profileService.me;
         var isAdmin = ((profileService.me.experiments || []).indexOf('admin') !== -1);
         scope.showFeatureUpsell =  isAdmin && (scope.me.orgs || []).filter(function(org) {
-            return org.slack && org.slack.slackTeams && org.slack.slackTeams.length > 0;
+            return org.slack && org.slack.slackTeam;
```
Comment (leogrim) [inline kifi-backend/modules/shoebox/angular/src/home/featureUpsell.js:19]: :ok_hand:

**Pull request merged**